client/src/App.js

```diff
@@ -18,7 +18,7 @@ import Container from './shared/components/Container';
 import HomePage from './shared/pages/Home';
 
 import Login from './cb-admin/pages/Login';
-import CbSignupPage from './cb-admin/pages/Signup';
+// import CbSignupPage from './cb-admin/pages/Signup';
```

Comment:
🔥

Reply:
Yeah I wasn't sure. Think we should just delete it? Don't mind.

Reply:
Yeah, I would. Otherwise it's just a bit of clutter we'll need to remove later.